src/main/java/net/dries007/tfc/mixin/client/FishingHookRendererMixin.java

```diff
@@ -21,7 +21,7 @@
 @Mixin(FishingHookRenderer.class)
 public abstract class FishingHookRendererMixin
 {
-    @Redirect(method = "render(Lnet/minecraft/world/entity/projectile/FishingHook;FFLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;I)V", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/item/ItemStack;is(Lnet/minecraft/world/item/Item;)Z"))
+    @Redirect(method = "render(Lnet/minecraft/world/entity/projectile/FishingHook;FFLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;I)V", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/item/ItemStack;is(Lnet/minecraft/world/item/Item;)Z"), require=0)
```

Comment:
Formatting, and please add a note explaining why require=0.

Also, why require=0 ?